theme/material/vaadin-radio-button-styles.ts

```diff
@@ -0,0 +1,4 @@
+import { registerStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import { radioButtonStyles } from './vaadin-radio-button-css';
+
+registerStyles('vaadin-radio-button', radioButtonStyles, { moduleId: 'lumo-radio-button' });
```

Comment:
`moduleId` should be updated to be `material`

Reply:
Done.